api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3107,16 +3111,24 @@ public void savePatientIdentifierType_shouldThrowErrorWhenTryingToSaveAPatientId
 	}
 	
 	@Test(expected = PatientIdentifierTypeLockedException.class)
-	public void retirePatientIdentifierType_shouldThrowErrorWhenTryingToRetireAPatientIdentifierTypeWhilePatientIdentifierTypesAreLocked()
-	    throws Exception {
+	public void retirePatientIdentifierType_shouldThrowErrorWhenTryingToRetireAPatientIdentifierTypeWhilePatientIdentifierTypesAreLocked() {
 		PatientService ps = Context.getPatientService();
 		createPatientIdentifierTypeLockedGPAndSetValue("true");
 		PatientIdentifierType pit = ps.getPatientIdentifierType(1);
 		ps.retirePatientIdentifierType(pit, "Retire test");
 	}
 	
 	@Test(expected = APIException.class)
```

Comment:
Can you please change this to use junit rules so that you can check the error message too

Reply:
Sure @wluyima . I will do it. 
Some other existing methods also need this changes. So I will cover those as well.